nucypher/policy/models.py

```diff
@@ -104,26 +106,32 @@ class Policy:
     and generates a TreasureMap for the Policy, recording which Ursulas got a KFrag.
     """
 
-    def __init__(self, alice, label, bob=None, kfrags=(constants.UNKNOWN_KFRAG,),
-                 public_key=None, m=None, alices_signature=constants.NOT_SIGNED) -> None:
+    def __init__(self,
+                 alice,
+                 label,
+                 bob=None,
+                 kfrags=(constants.UNKNOWN_KFRAG,),
+                 public_key=None,
+                 m: int = None,
+                 alices_signature=constants.NOT_SIGNED) -> None:
 
         """
         :param kfrags:  A list of KFrags to distribute per this Policy.
         :param label: The identity of the resource to which Bob is granted access.
         """
-        self.alice = alice
-        self.label = label
-        self.bob = bob
-        self.kfrags = kfrags
+        self.alice = alice                     # type: Alice
+        self.label = label                     # type: bytes
```

Comment:
Should these comments be type hints, or we want to keep them flexible?

Reply:
Something strange is going on - I don't even show this material on the diff.  If I click the filename above (`nucypher/policy/models.py`), I am taken to the diff page, but this file doesn't appear.  Am I the only one?

Reply:
...OK, so it looks like it's *not* actually part of this diff, as it was merged on Friday as 4c02e5cd9e6f41621ea3af696c9a4e31794f15f3 with #444.  

And no, a base-swap doesn't fix the issue.

Github, you're looking increasingly unstable.

Reply:
If I remember correctly, it was not shown as a part of the diff for me, but I still had an opportunity to comment